docs/production-deployment/cloud/introduction/security.mdx

```diff
@@ -80,9 +80,25 @@ If you are interested in leveraging AWS PrivateLink in your Namespaces, [create
 
 To set up PrivateLink connectivity with Temporal Cloud, you'll need to perform the following high-level steps:
 
-1. Add a VPC Endpoint Interface in your AWS VPC.
+1. Add a VPC Endpoint Interface in your AWS VPC using the below table.
 1. Configure security groups to allow TCP ingress traffic to port `7233` (for gRPC communication).
```

Comment:
See for the detailed steps with screenshots: https://docs.google.com/document/d/1gsFwTuheGevFZEunVGGM_M6XYvURqML1sY9K1h1IKKw/edit